chef/mountetna/resources/etna_app.rb

```diff
@@ -17,7 +17,7 @@
   file "/var/mountetna/#{name}/config.yml" do
     owner 'root'
     group 'docker'
-    content(EtnaConfigBuilder.new(name, node).merge(new_resource.extra_yml_config).build.to_yaml)
+    content(EtnaConfigBuilder.new(name, node, new_resource.tag).merge(new_resource.extra_yml_config).build.to_yaml)
```

Comment:
Not strictly necessary -- this is just a pre-emptive change to the future docker deployment that applies the config.yml change needed.

Reply:
I will have a chef repo change that adds the correct config.yml changes for polyphemus coming separately.

Reply:
May have to update the chef deploy script to also enable the apache service? At least currently the host doesn't resolve in my browser...

Reply:
Yeah, I'll have some work getting polyphemus webserver up.